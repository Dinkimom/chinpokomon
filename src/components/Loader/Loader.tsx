import React, { HTMLAttributes } from 'react';
import './Loader.css';

export const Loader: React.FC<HTMLAttributes<HTMLDivElement>> = ({
  className,
  ...props
}) => (
  <div className={`${className} loader`} {...props}>
    <p>Loading</p>

    <div className="loader__ring">
      <span></span>
    </div>
  </div>
);
