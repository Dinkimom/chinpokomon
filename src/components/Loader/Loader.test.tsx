import React from 'react';
import { shallow } from 'enzyme';

import { Loader } from './Loader';

describe('<Loader /> : initial', () => {
  it('should render correctly', () => {
    const component = shallow(<Loader />);

    expect(component).toMatchSnapshot();
  });
});
