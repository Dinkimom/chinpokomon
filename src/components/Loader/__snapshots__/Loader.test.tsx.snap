// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> : initial should render correctly 1`] = `ShallowWrapper {}`;
